src/metabase/query_analysis/native_query_analyzer/impl.cljc

```diff
@@ -51,7 +51,7 @@
      :quotes-preserve-case? (not (contains? #{:mysql :sqlserver} driver))
      :features              {:postgres-syntax        (isa? driver/hierarchy driver :postgres)
                              :square-bracket-quotes  (= :sqlserver driver)
-                             :unsupported-statements true
+                             :unsupported-statements false
```

Comment:
There's a surprising regression without this, where basic functions like COUNT suddenly return unsupported ASTs.

I don't understand why, and I'm not stepping into this rabbit hole.

Failing closed is probably the right choice in retrospect anyway.